lib/hawk/model/proxy.rb

```diff
@@ -80,7 +80,12 @@ def method_missing(meth, *args, &block)
           if klass.respond_to?(meth)
 
             method = klass.method(meth)
-            dsl_method = method.owner.parents.include?(Hawk::Model)
+            dsl_method =
+              if method.owner.respond_to?(:module_parents)
```

Comment:
not nice, can we define `module_parent` as `parent` if not
defined, and change these points just by referencing `module_parent` instead of `parent`?

thx

Reply:
Sorry @vjt, I'm not sure I've got the suggestion

`owner` may be `Class` or `Module` objects that could have not been defined by Hawk. `parents` and `module_parents` are monkey patch added by Active Support on top of Ruby's `Module`.

Hawk's specs only cover cases in which the method owner is `Hawk::Model::Querying::ClassMethods` and `Hawk::Model` is always present in the test cases. When running tests on a monolithic Rails application using Hawk, this method will check for `owners` whose the only parent is `Object`

Reply:
The suggestion (if applicable) would be us to define `module_parent` if not defined yet by Rails. Would that work?